pkg/controllers/management/nodepool/nodepool.go

```diff
@@ -251,7 +266,7 @@ func (c *Controller) createOrCheckNodes(nodePool *v3.NodePool, simulate bool) (b
 
 		changed = true
 		if !simulate {
-			c.deleteNode(toDelete, 0)
+			c.deleteNode(toDelete, deletionTime)
```

Comment:
This is just doing to delay the delete, if the intention is to stop a delete we need to move this